tests/api/test_bibliotheca.py

```diff
@@ -684,18 +685,23 @@ def test_findaway_license_to_webpub_manifest(
         assert 16.201 == first["duration"]
         assert "Track 1" == first["title"]
 
-        # There is no 'href' value for the readingOrder items because the
+        # The 'href' value for the readingOrder is a URN because the
```

Comment:
Minor: This first paragraph could be taken to imply that the URN is used by the Findaway SDK. 

Possible rephrasing:
> The 'href' value for the `readingOrder` is a URN. We do not use it to retrieve files, since that must be done through the Findaway SDK.